ants/utils/process_args.py

```diff
@@ -20,6 +24,24 @@
     "double": "D",
 }
 
+temporary_folder = None
+
+def temp_dir():
+    global temporary_folder
+    if temporary_folder is None or not os.path.exists(temporary_folder):
+        temporary_folder = tempfile.mkdtemp(prefix="ANTsPyTmp")
+    return temporary_folder
+
+def temp_file(suffix=None, prefix=None, dir=None):
+    if dir is None:
+        dir = temp_dir()
+    return tempfile.mktemp(suffix=suffix, prefix=prefix, dir=dir)
```

Comment:
tempfile.mktemp is deprecated and may lead to naming collisions

https://docs.python.org/3/library/tempfile.html#deprecated-functions-and-variables

Reply:
Will fix this week